src/components/VirtualList/README.md

```diff
@@ -0,0 +1,188 @@
+Render large lists by using virtualization to draw only the visible and overscan items.
+
+Configurations:
+
+```js
+class Example extends React.Component {
```

Comment:
I wonder if it's worth extracting out these heavy examples into dedicated example components; especially considering how complex the render method is.